docs/quickstart.rst

```diff
@@ -36,73 +26,64 @@ Let's load version 1.4.1 of the emodb_ database.
-  columns:
-    emotion: {scheme_id: emotion, rater_id: gold}
-    emotion.confidence: {scheme_id: confidence, rater_id: gold}
-emotion.categories.test.gold_standard:
-  type: filewise
-  split_id: test
-  columns:
-    emotion: {scheme_id: emotion, rater_id: gold}
-    emotion.confidence: {scheme_id: confidence, rater_id: gold}
-emotion.categories.train.gold_standard:
-  type: filewise
-  split_id: train
-  columns:
-    emotion: {scheme_id: emotion, rater_id: gold}
-    emotion.confidence: {scheme_id: confidence, rater_id: gold}
-files:
-  type: filewise
-  columns:
-    duration: {scheme_id: duration}
-    speaker: {scheme_id: speaker}
-    transcription: {scheme_id: transcription}
-
-Each table contains columns (:class:`audformat.Column`)
-that have corresponding schemes (:class:`audformat.Scheme`)
-describing its content.
-For example,
-to get an idea about the emotion annotations
-stored in the ``emotion`` column,
-we can inspect the corresponding scheme.
-
->>> db.schemes["emotion"]
-description: Six basic emotions and neutral.
-dtype: str
-labels: [anger, boredom, disgust, fear, happiness, sadness, neutral]
-
-Finally, we get the actual annotations
-as a :class:`pandas.DataFrame`.
-
->>> df = db["emotion"].get()  # get table
->>> df[:3]  # show first three entries
-                                           emotion  emotion.confidence
+>>> db = audb.load("emodb", version="1.4.1")
+
+Inspect label schemes,
+and request emotion labels for the test split.
+
+>>> list(db.schemes)
+['age',
+ 'confidence',
+ 'duration',
+ 'emotion',
+ 'gender',
+ 'language',
+ 'speaker',
+ 'transcription']
+>>> db.get("emotion", splits="test")  # returns dataframe
+                       emotion
+file
+.../wav/12a01Fb.wav  happiness
+.../wav/12a01Lb.wav    boredom
+.../wav/12a01Nb.wav    neutral
+.../wav/12a01Wc.wav      anger
+.../wav/12a02Ac.wav       fear
+...                        ...
+
+Or inspect tables,
+and request labels from a table.
+
+>>> list(db)
+['emotion',
+ 'emotion.categories.test.gold_standard',
+ 'emotion.categories.train.gold_standard',
+ 'files',
+ 'speaker']
+>>> db["emotion.categories.test.gold_standard"].get()  # returns dataframe
+                       emotion  emotion.confidence
 file
-...emodb/1.4.1/d3b62a9b/wav/03a01Fa.wav  happiness                0.90
-...emodb/1.4.1/d3b62a9b/wav/03a01Nc.wav    neutral                1.00
-...emodb/1.4.1/d3b62a9b/wav/03a01Wa.wav      anger                0.95
+.../wav/12a01Fb.wav  happiness                0.95
+.../wav/12a01Lb.wav    boredom                0.90
+.../wav/12a01Nb.wav    neutral                0.95
+.../wav/12a01Wc.wav      anger                0.95
+.../wav/12a02Ac.wav       fear                0.90
+...                        ...                 ...
+
+Load a media file,
+selected from the index of the dataframe
+or from the files index ``db.files``.
+
+.. skip: start
+
+>>> import audiofile
+>>> signal, sampling_rate = audiofile.read(db.files[0], always_2d=True)
```

Comment:
signal, sampling_rate = audiofile.read(db.files[0], always_2d=True) # in case the file is stereo

Reply:
I just realized that this is not needed as `signal.T` also works for signals with a shape of `(n,)`.
`always_2d=True` enforces mono signals to also have a channel dimension, e.g. `(1, n)`.
If it is `False` it returns `(n,)`.
For multi-channel with c channels it will always return the signal in the form `(c, n)`.

I updated the example now to:

```python
>>> signal, sampling_rate = audiofile.read(db.files[0])
```